src/devices/Ssd13xx/Commands/Ssd1327Commands/SetDisplayClockDivideRatioOscillatorFrequency.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Ssd13xx.Commands.Ssd1327Commands
+{
+    public class SetDisplayClockDivideRatioOscillatorFrequency : ISsd1327Command
+    {
+        /// <summary>
+        /// This command sets the divide ratio to generate DCLK (Display Clock) from CLK and
+        /// programs the oscillator frequency Fosc that is the source of CLK if CLS pin is pulled high.
+        /// </summary>
+        /// <param name="displayClockDivideRatio">Display clock divide ratio with a range of 0-15.</param>
```

Comment:
this should also have some info on the unit - I initially thought the value can be used directly but obviously divider `0` does not make sense which suggests it means something else (if this is infinity/disable clock please mention that)

Reply:
According to datasheet 0 stands for 1. But I'm sure we sould not map this values by ourselvs to meet datasheet.